src/main/kotlin/com/terraformation/backend/search/SerachExceptions.kt

```diff
@@ -0,0 +1,8 @@
+package com.terraformation.backend.search
```

Comment:
File naming nit: "Search" is misspelled. But luckily it doesn't matter: our convention for source files with multiple exception classes is to call them `Exceptions.kt`.

Reply:
Also, maybe this exception belongs in the PR that adds the code that checks for exportability -- as far as I can tell, it's unused at this point.

Reply:
Forget my last comment. I overlooked the change that throws this exception.